stable/mongodb/README.md

```diff
@@ -96,6 +96,8 @@ The following table lists the configurable parameters of the MongoDB chart and t
 | `replicaSet.pdb.minAvailable.arbiter`              | PDB (min available) for the MongoDB Arbiter nodes                                                                                                         | `1`                                                      |
 | `replicaSet.pdb.maxUnavailable.secondary`          | PDB (max unavailable) for the MongoDB Secondary nodes                                                                                                     | `nil`                                                    |
 | `replicaSet.pdb.maxUnavailable.arbiter`            | PDB (max unavailable) for the MongoDB Arbiter nodes                                                                                                       | `nil`                                                    |
+| `annotations`                                      | Annotations to be added to the deployment or statefulsets                                                                                                 | `{}`                                                     |
+| `labels`                                           | Additional labels for the deployment or statefulsets                                                                                                      | `{}`                                                     |
```

Comment:
Should we name these parameters `additionalLabels` and `additionalAnnotations`, respectively?

Reply:
We can, if you prefer. I find `labels` & `annotations` shorter and easier to use, and these names are already used on lots of existing charts.

Reply:
It's a minor thing. I don't have a strong opinion against `labels` & `annotations`